shared/fixes/ansible/no_rsh_trust_files.yml

```diff
@@ -16,7 +16,7 @@
       file:
           path: "{{ item.path }}"
           state: absent
-      with_items: "{{ shosts_equiv_locations }}"
-  when: shosts_equiv_locations
+      with_items: "{{ shosts_equiv_locations.files }}"
```

Comment:
I don't understand the fix. Isn't this the variable set by task above `"Detect shosts.equiv Files on the System"`, on line 13? Why are we changing this?

Reply:
Because `find` stores results in the list `.files`

http://docs.ansible.com/ansible/latest/find_module.html
Good catch, @WesleyCeraso